distributed/client.py

```diff
@@ -3850,7 +3852,13 @@ def collections_to_dsk(collections, *args, **kwargs):
         return collections_to_dsk(collections, *args, **kwargs)
 
     def get_task_stream(
-        self, start=None, stop=None, count=None, plot=False, filename="task-stream.html"
+        self,
+        start=None,
+        stop=None,
+        count=None,
+        plot=False,
+        filename="task-stream.html",
+        resources=INLINE,
```

Comment:
Could you please change this `bokeh_resources` to avoid confusion with the `resources` parameter that's commonly used throughout the codebase for specifying worker resources

Reply:
IIUC the default value for `resources` can be configured using [environment variables](https://docs.bokeh.org/en/latest/docs/reference/settings.html#resources). Do you know if passing `INLINE` here will take priority over any user-configured environment variables? 

If so, we could use `bokeh_resources=None` as the default value here. That way bokeh resources can still be configured from `get_task_stream`, and we continue to fallback to `bokeh`'s default behavior, but we don't override user-configured `bokeh` environment variables.